### PR TITLE
add documentation for a new VM

### PR DESCRIPTION
**Pull request by kayiwa**
these steps help to create a VM the hard way. Automation coming soon.

Co-authored-by: Vickie Karasic <vickiekarasic@users.noreply.github.com>

### CODE REVIEW

File: projects/create_new_vm.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+2. Put in an l-support ticket to open firewall rules (assign to yourself)
+3. Log into Palo Alto and [create rules](https://github.com/pulibrary/pul-it-handbook/blob/main/services/panos_fw.md)
+4. Respond to the ticket with details of the rules and close ticket (see [examples](https://github.com/pulibrary/ops-catchall/blob/main/projects/panos_editing.md) of ServiceNow ticket wording)
+
+
+## Prepare your New VM
+
+- Log in to new VM
+-  - Because this is the first time logging into the VM, you may get a warning about a man-in-the-middle attack, and you will need to edit the .ssh/known_hosts file to add your ssh keys at the line specified in the output you receive. Shortcut command to do this: ```vim ~/.ssh/known_hosts +"[line#]d|x"```
+        - Change the name of the VM:
```
Comment (acozine) [inline projects/create_new_vm.md:30]: The `common` role does this automatically, so we do not NEED to do this, but it's good to have documentation anyway.

File: projects/create_new_vm.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            ```sudo perl -pi -e s/lib-vm/[VM name]/g /etc/hosts```
+            ```sudo perl -pi -e s/lib-vm/[VM name]/g /etc/hostname```
+        - ```sudo apt -y update```
+        - ```sudo apt -y upgrade```
+        - ```sudo /sbin/reboot```
+
+## Expand the disk size
+
+Our VMS will be using 14GB or the allocated thin provisioned 28GB. The following steps will expand it to utilize all of the space. The following steps will expand your initial disk size
```
Comment (acozine) [inline projects/create_new_vm.md:39]: This isn't a comment on the current PR, but . . . if we're expanding the disk size every time, do we want to stop using thin provisioning?

**Review by acozine — COMMENTED** [review]
Overall looks great. Couple suggestions and questions.

File: projects/create_new_vm.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- ```sudo lvextend -l +100%FREE /dev/ubuntu-vg/ubuntu-lv```
+- ```sudo resize2fs /dev/ubuntu-vg/ubuntu-lv```
+- ```sudo /sbin/reboot```
```
Comment (VickieKarasic) [inline projects/create_new_vm.md:43]:
```suggestion
```

**Pull request merged**